testing_app/integration_test/app_test.dart

```diff
@@ -80,7 +80,7 @@ void main() {
 
       // Tap on the remove icon.
       await tester.tap(find.byKey(ValueKey('remove_icon_5')));
-      await tester.pumpAndSettle();
+      await tester.pumpAndSettle(Duration(seconds: 1));
```

Comment:
Adding `Duration` to `pumpAndSettle` after performing every action was required because ScaffoldMessenger keeps Snackbars from previous screens.